aicsdeformation/loaders/czi_movie_loader.py

```diff
@@ -0,0 +1,206 @@
+import cv2
+import numpy as np
+from pathlib import Path
+from typing import List, Dict, Optional
+
+from imageio import imwrite
+from aicsimageio import AICSImage
+
+from .helpers import raise_a_if_b
+from .exception import MinBeadMatchNotMetException, InsufficientTimePointsException
+from .loader_abc import LoaderABC
+from .path_images import PathImages
+
+
+NpImage = np.ndarray  # (Y, X) data image
+Dcube = np.ndarray  # (Z, Y, X) data cube
+Matrix = np.ndarray
+Char = str  # string of length 1
+SIFT_Type = cv2.xfeatures2d_SIFT
+
+
+class CziMoveLoader(LoaderABC):
```

Comment:
I think you meant to spell `CZIMovieLoader` (missing the i in Movie)

That said, I think this should really be called `CZITimelapseLoader`

Reply:
I know I am picky about names.